shared/constants/chat2/index.js

```diff
@@ -171,8 +172,12 @@ export const isTeamConversationSelected = (state: TypedState, teamname: string)
   return meta.teamname === teamname
 }
 export const isInfoPanelOpen = (state: TypedState) => {
-  const routePath = getPath(state.routeTree.routeState, [chatTab])
-  return routePath.size === 3 && routePath.get(2) === 'chatInfoPanel'
+  if (flags.useNewRouter) {
+    return state.chat2.infoPanelOpen
```

Comment:
you can derive this from the stack with new router too, teams does [something similar](https://github.com/keybase/client/blob/master/shared/constants/teams.js#L297)

Reply:
I haven't checked out this branch, but is it known that the “Jump to...” and new conversation button are on a non-white background?

Reply:
I didn't notice that @adamjspooner good eye!